packages/button/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.2.0",
   "main": "dist/main.js",
   "license": "MIT",
+  "typings": "definitions/index.d.ts",
```

Comment:
please add it also to template https://github.com/wc-catalogue/blaze-elements/blob/master/toolbelt/templates/package.json